### ISSUE TITLE
Contradiction detection, long texts

### ISSUE THREAD

**Issue by Lameus** [expertspec/expert#64]
Now dealing with long texts (>512 tokens) is implemented with the averaging of logits for chunks obtained after text splitting (function "averaging from [core](https://github.com/expertspec/expert/tree/main/expert/core)/[contradiction](https://github.com/expertspec/expert/tree/main/expert/core/contradiction)/[contr_tools](https://github.com/expertspec/expert/tree/main/expert/core/contradiction/contr_tools)
/model_tools.py). However, it would be correct to average embeddings of chunks (sentence embeddings).
